sage_modabvar/abvar.py

```diff
@@ -590,7 +590,12 @@ def elliptic_curve(self):
         Return an elliptic curve isogenous to self. If self is not dimension 1
         with rational base ring, raise a ValueError.
 
-        The elliptic curve is found by looking it up in Cremona's tables.
+        The elliptic curve is found by looking it up in Cremona's tables. The
+        default mini-CremonaDatabase contains curves up to conductor 10000 and
+        the optional full CremonaDatabase contains curves up to conductor
+        379998. If a curve is not found in the CremonaDatabase, a RuntimeError
```

Comment:
Please  do not include the statement "the optional full CremonaDatabase contains curves up to conductor 379998" since that upper bound is constantly changing, and there is absolutely no way you can keep the statement in the docs here in sync with reality.  There is no value in so intentionally setting your docs up to be wrong.     Even the mini database could change....   So be more vague or use "over 300000" or something like that.